tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -959,7 +959,7 @@ def test_task_file_not_in_glacier(self,
         }
         mock_s3_cli = mock_boto3_client('s3')
         # todo: Verify the below with a real-world db. If not the same, fix request_files.object_exists
-        mock_s3_cli.head_object.side_effect = [ClientError({'Error': {'Code': 'NotFound'}}, 'head_object')]
+        mock_s3_cli.head_object.side_effect = [ClientError({'Error': {'Code': '404', 'Message': 'Not Found'}}, 'head_object')]
```

Comment:
Is Not Found in message supposed to have a space or should there be no space like NotFound?

Reply:
I am seeing "Not Found" in the error message.

{"message": "An error occurred (404) when calling the HeadObject operation: Not Found", "timestamp": "2021-05-24T14:28:07.521580", "level": "error"} {'Error': {'Code': '404', 'Message': 'Not Found'}